src/server/services/cms-content.ts

```diff
@@ -211,14 +210,17 @@ async function getGeneralPage(
           sanitizeCmsContent(responseData.applicatie.inhoud.tekst),
       };
     },
+    formatUrl({ url }) {
+      const urls = {
+        private: `${url}/ziet-amsterdam/?AppIdt=app-data`,
+        'private-attributes': '', // Not used
+        commercial: `${url}/overzicht-producten-ondernemers/?AppIdt=app-data`,
+      };
+      return urls[profileType] || urls.private;
+    },
```

Comment:
Ik snap dit blok niet zo goed, dus een paar vragen erover.
- Waarom is private attributes defined als het een lege string is en nooit wordt gebruikt? Kan deze regel niet weg?
- Hoezo defaulten we naar een private url? Kan dat niet de verkeerde conclusie zijn als we niet weten wat de profileType is?

Misschien dat een comment handig is hier met de redenen want het is niet standaard dat we geen profileType hebben toch?
En een idee, misschien is het makkelijker werken om aan het begin van de keten profileType al te setten als deze niet bestaat. Als je bijvoorbeeld meer logica bij de caller toevoegd en daar weer een profiletype moet hebben, kan je dit missen.

Reply:
ProfileType kan ook ‘private-attributes’ zijn. Ik heb de code wat versimpeld.